<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AVEditWidget</class>
 <widget class="QWidget" name="AVEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="m_pVertLayout">
    <item>
     <widget class="QWidget" name="m_pWidgetUpper" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>250</height>
       </size>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-1</y>
         <width>691</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="m_pHLayoutUpper">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="m_pTabWidgetSrc">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="m_pScrollAreaAVStream">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>687</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
